storage/storage-interface/src/state_store/state_view/db_state_view.rs

```diff
@@ -47,8 +49,18 @@ impl DbStateView {
 impl TStateView for DbStateView {
     type Key = StateKey;
 
-    fn get_state_value(&self, state_key: &StateKey) -> StateViewResult<Option<StateValue>> {
-        self.get(state_key)
+    fn id(&self) -> StateViewId {
+        if let Some(version) = self.version {
+            StateViewId::TransactionValidation {
```

Comment:
Is `DbStateView` only used for validation? Not sure I understand why version being `Some` means it is a validation view

Reply:
I don't think anyone is using the state view id right now, I probably should re-name this to be TransactionVersion to avoid confusion